spec/models/legislator_spec.rb

```diff
@@ -236,7 +240,7 @@
     end
 
     context "with args" do
-      subject(:response) { senator.as_json(extra_key: 'foo') }
+      subject(:response) { senator.as_json(extras: { extra_key: 'foo' }) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.